src/core/mainloop/mainloop.c

```diff
@@ -1454,6 +1457,7 @@ static periodic_event_item_t *fetch_networkstatus_event=NULL;
 static periodic_event_item_t *launch_descriptor_fetches_event=NULL;
 static periodic_event_item_t *check_dns_honesty_event=NULL;
 static periodic_event_item_t *save_state_event=NULL;
+static periodic_event_item_t *prune_old_routers_event=NULL;
```

Comment:
Do we need this?  NAMED_CALLBACK and callbacks here are only for the callbacks that we want to turn on and off by hand.